cma/evolution_strategy.py

```diff
@@ -4646,4 +4661,35 @@ def update(es):
     es.objective_function_complements = [_al]
     es.augmented_lagrangian = _al
     es.best_feasible = best_feasible_solution
+
+    if post_optimization:
+        if h != no_constraints and (not isinstance(post_optimization, float) or post_optimization <= 0):
```

Comment:
Python coding guidelines suggest to keep lines below 80 characters and they prefer braces over the backslash line continuation. See 

https://www.python.org/dev/peps/pep-0008/#maximum-line-length
https://google.github.io/styleguide/pyguide.html#32-line-length

Both style guides are well worth to be read entirely.